Gemfile

```diff
@@ -28,7 +28,9 @@ gem 'haml-rails', '~> 0.9.0'
 
 gem 'mysql2', '~> 0.4.4'
 gem 'devise', '~> 4.2.0'
-
+gem 'will_paginate', '~> 3.1.0'
```

Comment:
try looking at kirmani also. Its quite similar to this, but we are using that one in production